### PR TITLE
feat(frontend): new government masthead design

### LINKED ISSUE #1272: Add new government masthead
Enhanced design includes guidelines on how to visit and transact safely with government websites

https://www.figma.com/file/UE8eUTKfk8OaqPr6vegPIy/gov.sg-Masthead?node-id=0%3A1

### PR DESCRIPTION
**Pull request by lamkeewei**
## Problem

Hosted at https://masthead.postman.gov.sg for review.

Closes #1272 

## Solution

**Features**:

- Implemented new masthead design with explainers

## Before & After Screenshots
**Desktop**

image

image

**Mobile**

image

image

### CODE REVIEW

File: frontend/src/components/landing/banner/Banner.tsx [hunk 1]
```diff
@@ -1,16 +1,73 @@
...
+              </div>
+              <div>
+                Government agencies communicate via <strong>.gov.sg</strong>{' '}
+                websites (e.g. go.gov.sg/open).{' '}
+                <a
+                  href="http://www.go.gov.sg/trusted-websites"
+                  target="_blank"
+                  rel="noopener noreferrer"
+                >
+                  Trusted websites. <i className="bx bx-link-external" />
```
Comment (mjgui) [inline frontend/src/components/landing/banner/Banner.tsx:50]:
```suggestion
                  Trusted websites <i className="bx bx-link-external" />
```

File: frontend/src/styles/base/_typography.scss [hunk 2]
```diff
@@ -1,6 +1,7 @@
 // Fonts
 @import url('https://fonts.googleapis.com/css2?family=Noto+Sans+JP:wght@300;400;500;700&display=swap');
 @import url('https://fonts.googleapis.com/css2?family=Roboto+Mono&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Lato&display=swap');
```
Comment (mjgui) [inline frontend/src/styles/base/_typography.scss:4]: seems quite wasteful to import a whole font just for the banner, but not sure what the designers' take is on this?
(additionally, maybe we should just import specific weights to reduce pageload times)

**Review by mjgui — APPROVED** [review]
LGTM - pending comments from viv I guess

File: frontend/src/styles/base/_typography.scss (hunk 2 above)
Comment (lamkeewei) [reply frontend/src/styles/base/_typography.scss:4]: Yeah, not sure if `wght` is required for the default weight. Got the import statement from [here](https://fonts.google.com/specimen/Lato).

**Comment by viviennechong** [comment]
Looks good overall!

I just finished documenting the specifications for the dimensions of the icons/text divs and also the typography. (for desktop version) You can find these in the same [Figma file](https://www.figma.com/file/UE8eUTKfk8OaqPr6vegPIy/?node-id=0%3A1).

I think most important change is the text copy portion where the line breaks are specified. 

Preview of the file:
image

Thanks so much!! Appreciate ur work as always. 

Lmk if u have any questions

**Pull request merged**